src/components/common/deleteResource.jsx

```diff
@@ -70,7 +75,15 @@ export class DeleteResourceModal extends React.Component {
                            </Button>
                        </>
                    }>
-                { actionDescription || cockpit.format(_("Confirm this action")) }
+                <DescriptionList isHorizontal>
+                    {actionDescription || cockpit.format(_("Confirm this action"))}
```

Comment:
The designs looked like the "Confirm this action" string should disappear, and be replaced with something like "These ... will be removed:"?

Reply:
Or, I suppose that's just a fallback, and this will be cleaned up in #652?